ProcessMaker/processmaker
# Add Global Signals API Documentation

## Issue & Reproduction Steps
As a process architect I want to have documentation of the global signals API, so I can manage global signals

## Solution
- Add API documentation to Swagger

## How to Test
- Navigate to /api/documentation in your local instance.
- Authorize yourself via Bearer Token
- Locate the 'Signals' location
- 'Try Out' both APIs
- Ensure all fields work properly
- Ensure all properties are listed in the example schema

## Related Tickets & Packages
- [FOUR-5985](https://processmaker.atlassian.net/browse/FOUR-5985)

## Code Review Checklist
